tests/rohf-orb-rot/input.dat

```diff
@@ -0,0 +1,122 @@
+#! Test if DGAS's orbital rotation code matches expected values
+
+# Restrict occupations to compute lowest A'' state (X state)
+set { 
+    basis 6-31G
+    reference rohf
+    maxiter 500
+    e_convergence 10
+    d_convergence 10
+    r_convergence 10
+    memory 60000
+    freeze_core True
+    scf_type pk
+    docc [7, 1]
+    socc [0, 1]
+}
+
+molecule ho2 {
+    0 2
+    O -0.6937342972  0.0081785728  0.0000000000
+    O  0.6354757377 -0.0626608366  0.0000000000
+    H  0.9246056151  0.8646730647  0.0000000000
+}
+
+scf_e, scf_wfn = energy('scf', return_wfn=True)
+ccsd_e, ccsd_wfn = energy('ccsd', return_wfn=True)
+
+compare_values(-150.108136116503147, scf_e, 6, 'X SCF energy') #TEST
+compare_values(-150.347533153677489, ccsd_e, 6, 'X CCSD energy') #TEST
+
+clean()
+
+# Restrict occupations to compute lowest A' state (A state)
+set {
+    docc [6, 2]
+    socc [1, 0]
+}
+
+scf_e, scf_wfn = energy('scf', return_wfn=True)
+ccsd_e, ccsd_wfn = energy('ccsd', return_wfn=True)
+
+compare_values(-150.087298715913619, scf_e, 6, 'A SCF energy') #TEST
+compare_values(-150.312480530946573, ccsd_e, 6, 'A CCSD energy') #TEST
+
+clean()
+
+# Compute X state with symmetry turned off
+set {
+    docc [8]
+    socc [1]
+}
+
+molecule ho2 {
+    0 2
+    O -0.6937342972  0.0081785728  0.0000000000
+    O  0.6354757377 -0.0626608366  0.0000000000
+    H  0.9246056151  0.8646730647  0.0000000000
+    symmetry c1
+}
+
+scf_e, scf_wfn = energy('scf', return_wfn=True)
+ccsd_e, ccsd_wfn = energy('ccsd', return_wfn=True)
+
+compare_values(-150.108136116503147, scf_e, 6, 'X SCF energy') #TEST
+compare_values(-150.347533153677489, ccsd_e, 6, 'X CCSD energy') #TEST
+
+# Rotate orbitals to obtain A state in C1
+# Quick orbital rotation code
+def orb_rotate(C, h, i, j, theta):
```

Comment:
The goal of this test was more instructive than anything - if a user wished to learn how to perform an orbital rotation like this, there would be a reference on how to do so. I understand your rationale, though. Where would something like this example go instead? (Or is the thread concerning this in the forum helpful enough on its own?)